simulations/vip-499/bsctestnet.ts

```diff
@@ -0,0 +1,13 @@
+import { expectEvents } from "src/utils";
+import { forking, testVip } from "src/vip-framework";
+
+import { vip499 } from "../../vips/vip-499/bsctestnet";
+import OMNICHAIN_PROPOSAL_SENDER_ABI from "./abi/OmnichainProposalSender.json";
+
+forking(51177545, async () => {
+  testVip("VIP-499", await vip499(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [OMNICHAIN_PROPOSAL_SENDER_ABI], ["ExecuteRemoteProposal"], [2]);
```

Comment:
I would also check that `StorePayload` event is emitted 0 times (also on mainnet)